docker/Dockerfile.ci_qemu

```diff
@@ -64,3 +64,7 @@ RUN bash /install/ubuntu_install_qemu.sh
 COPY install/ubuntu_install_zephyr.sh /install/ubuntu_install_zephyr.sh
 RUN bash /install/ubuntu_install_zephyr.sh
 ENV ZEPHYR_BASE=/opt/zephyrproject/zephyr
+
+# Install ONNX
```

Comment:
I'm surprised this is needed. Why not just install the ONNX pip package?

Reply:
must reduce variation in ci-* images :)